e2e/components/InteractiveTag/InteractiveTag-test.avt.e2e.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+import { expect, test } from '@playwright/test';
+import { visitStory } from '../../test-utils/storybook';
+
+test.describe('@avt InteractiveTag', () => {
+  test('@avt-advanced-states DismissibleTag', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tag',
+      id: 'experimental-unstable-interactivetag--dismissible',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('DismissibleTag');
+  });
+
+  // Testing being skipped because it is failing in the ToggleTip that operational it's using
+  test.skip('@avt-advanced-states OperationalTag', async ({ page }) => {
```

Comment:
Do we still need to skip this test with the changes made to `ToggleTip`?

Reply:
Good catch! I just removed @tw15egan